Robot_Adapter/CRUD/Read/Loads/Loads.cs

```diff
@@ -259,6 +260,71 @@ private List<ILoad> ReadGeometricalLineLoads(List<string> ids = null)
+            for (int i = 0; i < lCases.Count; i++)
+            {
+                if (bhomLoadCases.ContainsKey(lCases[i].Name) == false)
+                    bhomLoadCases.Add(lCases[i].Name, lCases[i]);
+            }
+
+            //Dictionary<string, Loadcase> bhomLoadCases = ReadLoadCase().ToDictionary(x => x.Name);
+            IRobotCaseCollection loadCollection = m_RobotApplication.Project.Structure.Cases.GetAll();
+
+            for (int i = 1; i <= loadCollection.Count; i++)
+            {
+                IRobotCase lCase = loadCollection.Get(i) as IRobotCase;
+                if (lCase.Type == IRobotCaseType.I_CT_SIMPLE)
+                {
+                    IRobotSimpleCase sCase = lCase as IRobotSimpleCase;
+                    if (bhomLoadCases.ContainsKey(sCase.Name))
+                    {
+                        for (int j = 1; j <= sCase.Records.Count; j++)
+                        {
+                            IRobotLoadRecord loadRecord = sCase.Records.Get(j);
+                            List<int> elementIds = Convert.FromRobotSelectionString(loadRecord.Objects.ToText());
+                            List<Panel> objects = new List<Panel>();
+                            for (int k = 0; k < elementIds.Count; k++)
+                            {
+                                if (bhomPanel.ContainsKey(elementIds[k]))
+                                    objects.Add(bhomPanel[elementIds[k]]);
+                            }
+
+                            switch (loadRecord.Type)
+                            {
+                                case IRobotLoadRecordType.I_LRT_UNIFORM:
+                                    double fx = loadRecord.GetValue((short)IRobotUniformRecordValues.I_URV_PX);
+                                    double fy = loadRecord.GetValue((short)IRobotUniformRecordValues.I_URV_PY);
+                                    double fz = loadRecord.GetValue((short)IRobotUniformRecordValues.I_URV_PZ);
+                                    double ls = loadRecord.GetValue((short)IRobotUniformRecordValues.I_URV_LOCAL_SYSTEM);
+                                    double pj = loadRecord.GetValue((short)IRobotUniformRecordValues.I_URV_PROJECTED);
+
+
+                                    AreaUniformlyDistributedLoad PanelPressure = new AreaUniformlyDistributedLoad
+                                    {
+                                        Pressure = new Vector { X = fx, Y = fy, Z = fz },
+                                        Loadcase = bhomLoadCases[sCase.Name],
+                                        Objects = BH.Engine.Base.Create.BHoMGroup(objects) as BHoMGroup<IAreaElement>,
+                                        Axis = ls == 0 ? LoadAxis.Global : LoadAxis.Local,
+                                        Projected = pj == 1
+                                    };
+                                    bhomLoads.Add(PanelPressure);
+                                    break;
+                            }
```

Comment:
A switch should have a `default:` case, unsure of what happens when something else is provided, but we will want to log that

Reply:
I think it is fine to not have a deafult. I think it just does not do anything. Saying that, I think this might get more readable with just an if statement instead. (reason for the switch is that you will get passed all available loads from a case, and want to filter out only the area loads)